lib/menu.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'dish'
+require_relative 'formatter'
+
+class Menu
+  include Formatter
+
+  def initialize(dish_klass = Dish, dishes_str = nil)
+    @dish_klass = dish_klass
+    @dishes = Array.new
```

Comment:
Style/EmptyLiteral: Use array literal `[]` instead of `Array.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)